### ISSUE TITLE
Remove generated ioprojectatomicpodman.go from Git?

### ISSUE THREAD

**Issue by wking** [projectatomic/libpod#799]
We've tracked it since #669, but it's noisy:

```console
$ git log --oneline --stat=80 cmd/podman/varlink/ioprojectatomicpodman.go
d870c25 implement varlink commit
 cmd/podman/varlink/ioprojectatomicpodman.go | 1614 ++++++++++++++-------------
 1 file changed, 823 insertions(+), 791 deletions(-)
834f1f6 varlink info
 cmd/podman/varlink/ioprojectatomicpodman.go | 1741 +++++++++++++++------------
 1 file changed, 942 insertions(+), 799 deletions(-)
a74107b alphabetize the varlink methods, types, and errors in the docs
 cmd/podman/varlink/ioprojectatomicpodman.go | 1615 ++++++++++++++-------------
 1 file changed, 808 insertions(+), 807 deletions(-)
2526355 Generate varlink API documentation automatically
 cmd/podman/varlink/ioprojectatomicpodman.go | 2673 +++++++++++++++++----------
 1 file changed, 1700 insertions(+), 973 deletions(-)
8dfebd4 varlink containers
 cmd/podman/varlink/ioprojectatomicpodman.go | 1384 +++++++++++++++++----------
 1 file changed, 863 insertions(+), 521 deletions(-)
7a0a855 Fix misc stuff found by jhonce
 cmd/podman/varlink/ioprojectatomicpodman.go | 980 ++++++++++++++--------------
 1 file changed, 495 insertions(+), 485 deletions(-)
a824186 Use buildah commit and bud in podman
 cmd/podman/varlink/ioprojectatomicpodman.go | 972 ++++++++++++++--------------
 1 file changed, 486 insertions(+), 486 deletions(-)
39a7a77 varlink images
 cmd/podman/varlink/ioprojectatomicpodman.go | 1571 +++++++++++++++++++++++++++
 1 file changed, 1571 insertions(+)
```

because some entries seem to be randomly ordered (e.g. type definitions are not alphabetized).  Since I don't see docs anywhere claiming you can `go get github.com/projectatomic/libpod/pkg/podman`, can we drop it from version control and rely on [the existing `Makefile` rule][1] to ensure we have a fresh copy at build time?

cc @baude, since you've been doing all the regeneration work ;).

[1]: https://github.com/projectatomic/libpod/blob/4704c138ae7f3c98c3fddba2bc442d61f4eb1f28/Makefile#L265-L266

**Comment by mheon** [comment]
@baude I know there's a reason we checked it in, but I'm forgetting right now?

I would really love to see the Varlink generator changed to consistently order definitions so we cut down on the churn.

**Comment by baude** [comment]
we dont want this to be generated during package building.  it makes things too complicated.  given the noiseyness, a bug against varlink would be appropriate.

**Comment by jwhonce** [comment]
Closing, given @baude comment.

**Comment by wking** [comment]
Fixed by #840.
